src/api/app/models/token/workflow.rb

```diff
@@ -9,10 +9,10 @@ def self.token_name
 
   def call(options)
     set_triggered_at
+    @scm_webhook = options[:scm_webhook]
 
-    raise Token::Errors::MissingPayload, 'A payload is required' if options[:payload].nil?
+    raise Token::Errors::MissingPayload, 'A payload is required' if @scm_webhook&.payload.nil?
```

Comment:
`payload` is always going to be there since it's defined as a `attr_accessor`: https://github.com/openSUSE/open-build-service/blob/550f59cdb733d0634458e5e229e5b3f0afa20a5c/src/api/app/models/scm_webhook.rb#L5

So with this in mind, we don't need the safe navigation:
```suggestion
    raise Token::Errors::MissingPayload, 'A payload is required' if @scm_webhook.payload.nil?
```

Reply:
Some specs start to fail if we remove safe navigation.

Reply:
I pushed a fix for that.

Reply:
Perfect. I've squashed the new commit.